lab_1/main.py

```diff
@@ -1,42 +1,35 @@
-"""
-Lab 1
-A concordance extraction
-"""
+import re
+def tokenize(text: str) -> list:
+    if isinstance(text,str): #проверяем, строка ли это
+        return []
+    text = re.sub(r'[^A-Za-z 0-9]', '', text) # с помощью рег. выраж. убираю знаки препинания и числительные
+    tokenize_list = text.split().lower()
+    return tokenize_list
+pass
 
 
-def tokenize(text: str) -> list:
-    """
-    Splits sentences into tokens, converts the tokens into lowercase, removes punctuation
-    :param text: the initial text
-    :return: a list of lowercased tokens without punctuation
-    e.g. text = 'The weather is sunny, the man is happy.'
-    --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
-    """
-    pass
 
 
 def remove_stop_words(tokens: list, stop_words: list) -> list:
-    """
-    Removes stop words
-    :param tokens: a list of tokens
-    :param stop_words: a list of stop words
-    :return: a list of tokens without stop words
-    e.g. tokens = ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
-    stop_words = ['the', 'is']
-    --> ['weather', 'sunny', 'man', 'happy']
-    """
-    pass
+    if isinstance(tokenize_list, list) and isinstance(stop_words, list): #проверяем, списки ли это
+        for token in stop_words:
+            if token not in stop_words: #проверяем входит ли слово в список стоп-слов
+                tokenize_list.append(token)
+            return tokenize_list
+pass
 
 
 def calculate_frequencies(tokens: list) -> dict:
-    """
-    Calculates frequencies of given tokens
-    :param tokens: a list of tokens without stop words
-    :return: a dictionary with frequencies
-    e.g. tokens = ['weather', 'sunny', 'man', 'happy']
-    --> {'weather': 1, 'sunny': 1, 'man': 1, 'happy': 1}
-    """
-    pass
```

Comment:
Please, leave the description of the function how it is - use the already written template.